### ISSUE TITLE
Error:  'ClassificationTab' object has no attribute 'applyClassSymbologyVector'

### ISSUE THREAD

**Issue by merkato** [semiautomaticgit/SemiAutomaticClassificationPlugin#138]
Trying to do classification, discovered, that SCP interface is somewhat corrupted - has no tabs, where it should be. Additionally after starting classification I've get that error. Classification finished, but no vector can be created (if I cancel it, raster is ok).

```
AttributeError: 'ClassificationTab' object has no attribute 'applyClassSymbologyVector' 
Traceback (most recent call last):
  File "/home/mechanik/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin/maininterface/classificationTab.py", line 121, in runClassificationAction
    self.runClassification(algorithmFilesCheck = algFilesCheck, reportCheck = report, vectorConversion = vector, useMacroclass = macroclass, useLcs = useLcs, useLcsAlgorithm = useLcsAlgorithm, LCSLeaveUnclassified = leaveUnclassified, maskCheckBox = maskC, maskPath = maskPath)
  File "/home/mechanik/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin/maininterface/classificationTab.py", line 227, in runClassification
    self.applyClassSymbologyVector(vl, cfg.macroclassCheck, cfg.qmlFl, sL)
AttributeError: 'ClassificationTab' object has no attribute 'applyClassSymbologyVector'


Wersja Pythona: 3.8.6 (default, Sep 25 2020, 09:36:53) [GCC 10.2.0] 
Wersja QGIS: 3.16.1-Hannover Hannover, b381a90dca 


```
![Semi-Automatic Classification Plugin_003](https://user-images.githubusercontent.com/847295/100584157-00ade880-32ec-11eb-86ee-f7a25af3824f.png)

**Comment by semiautomaticgit** [comment]
Thank you, I'm going to fix this

**Comment by semiautomaticgit** [comment]
It should be fixed in version 7.2
